docs/modules/ROOT/pages/CHANGELOG.adoc

```diff
@@ -1,5 +1,35 @@
 = Changelog
 
+[[release-0.8.0]]
+== 0.8.0 (2024-07-04)
+
+=== Additions
+
+* Support for Pkl 0.26
+
+=== Fixes
+
+* Make the loading of `generator-settings.pkl` observe the CLI's `--cache-dir` for air-gapped use.
+* Fix decoding of sets when input type is any.
+
+=== Miscellaneous
+
+* Refactoring for performance and clarity of the parsing and comparison of versions.
+* Fix unhandled error in the evaluator manager.
+* Reduced lock contention in `atomicRandom`.
+* Resolution of the path to `PklProject` files is is made platform independent by using `path.Join`.
```

Comment:
```suggestion
* Use `path.join` to make the resolution of `PklProject` files platform independent.
```
If you decide to keep the wording as is, you should remove the double `is`: `files is is made...`.

Reply:
I tried to avoid putting an implementation detail (`path.Join`) so far towards the front... made the line that much more stilted. Thanks.